README.md

```diff
@@ -163,74 +158,64 @@ Abilities MUST NOT be case sensitive. For example, `http/post`, `http/POST`, `HT
 
 There MUST be at least one path segment as a namespace. For example, `http/put` and `db/put` MUST be treated as unique from each other.
 
-The only reserved ability MUST be the un-namespaced [`"*"` (superuser)](#41-superuser), which MUST be allowed on any resource.
-
-## 2.4 Capability
+The only reserved ability MUST be the un-namespaced [`"*"` (top)][top ability], which MUST be allowed on any resource.
 
-A capability is the association of an "ability" to a "resource": `resource x ability`.
+## 2.4 Caveats
 
-The `with` (resource) and `can` (ability) fields are REQUIRED. The `nb` (non-normative extension) field is OPTIONAL.
+Capabilities MAY define additional optional or required fields specific to their use case in the caveat fields. This field is OPTIONAL in the general case, but MAY be REQUIRED by particular capability types that require this information to validate. Caveats MAY function as an "escape hatch" for when a use case is not fully captured by the resource and ability fields. Caveats can be read as "on the condition that `<some caveat>` holds".
 
-``` json
-{
-  "with": $RESOURCE,
-  "can": $ABILITY,
-  "nb": $EXTENSION
-}
-```
+## 2.5 Capability
 
-### 2.4.1 `nb` Non-Normative Fields
+A capability is the association of an "ability" to a "resource": `resource x ability x caveats`.
 
-Capabilities MAY define additional optional or required fields specific to their use case in the `nb` ([nota bene](https://en.wikipedia.org/wiki/Nota_bene)) field. This field is OPTIONAL in the general case, but MAY be REQUIRED by particular capability types that require this information to validate. The `nb` field MAY contain additional caveats or other important information related to specifying the capability, and MAY function as an "escape hatch" for when a use case is not fully captured by the `with` and `can` fields.
+The resource and ability fields are REQUIRED. Any non-normative extensions are OPTIONAL.
 
-Further delegation of a capability with `nb` fields set MUST respect the `nb` fields. On validation, if a `nb` field is present, it MUST be checked. If a validator is not able to interpret the `nb` field, it MUST reject the capability. As such, any `nb` caveats from a proof MUST further attenuate the delegated capability.
+```
+{ $RESOURCE: { $ABILITY: [ $CAVEATS ] } }
+```
 
-#### 2.4.2 Examples
+### 2.5.1 Examples
 
 ``` json
-[
-  {
-    "with": "example://example.com/public/photos/",
-    "can": "crud/delete"
-  },
-  {
-    "with": "example://example.com/private/84MZ7aqwKn7sNiMGsSbaxsEa6EPnQLoKYbXByxNBrCEr",
-    "can": "wnfs/append"
-  },
-  {
-    "with": "example://example.com/public/photos/",
-    "can": "crud/delete",
-    "nb": {
-      "matching": "/(?i)(\W|^)(baloney|darn|drat|fooey|gosh\sdarnit|heck)(\W|$)/"
-    }
+{
+  "example://example.com/public/photos/": {
+    "crud/read": [{}],
```

Comment:
Yeah I agree that this is confusing initially. I'm pretty sure that this syntax leaves open a couple ambiguities that have to get clarified in prose.

> is it more common to see AND-ed caveats than OR-ed caveats?

It is more common in contexts where you have a single capability at a time (which is arguably cleaner but less convenient especially in a passive setting)! The tradeoff is that it makes the syntax less expressive, because with an AND you can't express disjoint capabilities. I'm also unsure that your average programmer would know if it was an AND or an OR on first inspection — it's ambiguous in a way that the 0.9 syntax is not.